rotkehlchen/api/v1/schemas.py

```diff
@@ -520,10 +519,9 @@ def make_asset_movements_query(  # pylint: disable=no-self-use
         }
 
 
-class TimerangeQuerySchema(Schema):
+class TimerangeQuerySchema(AsyncQueryArgumentSchema):
```

Comment:
This change will move the `async_query` argument in the keyword arguments of wherever `TimerangeQuerySchema` used from the end to the start, so the functions it's used in should also be modified or else they will fail.


Edit: Hm ... I checked the code and I don't see `TimerangeQuerySchema` used anywhere. You can delete the entire Schema.